app/templates/macros/nav_macros.html

```diff
@@ -28,26 +28,25 @@
     {% set calendar = [] %}
     {% set resources = [] %}
     {% set scattergram = [] %}
+    {% set colleges = [] %}
     {% if current_user.is_authenticated %}
       {% set user = ([(current_user.role.index + '.index', current_user.role.name + ' Dashboard', 'user')]) %}
       {% set calendar = ([('main.calendar', 'Calendar', 'info')]) %}
       {% set scattergram = ([('main.scattergram', 'Scattegram', 'info')]) %}
```

Comment:
Let's change "Scattegram" to "Colleges" and have this tab navigate to the college database page.